server/valgrind-out.txt

```diff
@@ -1,117 +1,137 @@
+--7744-- REDIR: 0x493c930 (libc.so.6:wcslen) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x491fae0 (libc.so.6:strspn) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x4920750 (libc.so.6:stpncpy) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x49206f0 (libc.so.6:stpcpy) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x4921c90 (libc.so.6:strchrnul) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x49208a0 (libc.so.6:strncasecmp_l) redirected to 0x48311d0 (_vgnU_ifunc_wrapper)
+--7744-- REDIR: 0x49318c0 (libc.so.6:__strrchr_sse2) redirected to 0x483ea70 (__strrchr_sse2)
+--7744-- REDIR: 0x491a0e0 (libc.so.6:malloc) redirected to 0x483b780 (malloc)
+--7744-- REDIR: 0x491bb10 (libc.so.6:calloc) redirected to 0x483dce0 (calloc)
+--7744-- REDIR: 0x49316b0 (libc.so.6:__strchrnul_sse2) redirected to 0x4843540 (strchrnul)
+--7744-- REDIR: 0x4931bc0 (libc.so.6:__strlen_sse2) redirected to 0x483efa0 (__strlen_sse2)
+--7744-- REDIR: 0x491a6d0 (libc.so.6:free) redirected to 0x483c9d0 (free)
+--7744-- REDIR: 0x493bad0 (libc.so.6:memcpy@GLIBC_2.2.5) redirected to 0x4840100 (memcpy@GLIBC_2.2.5)
+--7744-- REDIR: 0x491ffa0 (libc.so.6:__GI_strstr) redirected to 0x4843ca0 (__strstr_sse2)
+--7744-- REDIR: 0x4a0d790 (libc.so.6:__memcmp_sse4_1) redirected to 0x4842150 (__memcmp_sse4_1)
+--7744-- REDIR: 0x492a3b0 (libc.so.6:__strcmp_sse2_unaligned) redirected to 0x483fed0 (strcmp)
+--7744-- REDIR: 0x493bab0 (libc.so.6:__mempcpy_sse2_unaligned) redirected to 0x4843660 (mempcpy)
+--7744-- REDIR: 0x491ae80 (libc.so.6:realloc) redirected to 0x483df30 (realloc)
+--7744-- REDIR: 0x493bf00 (libc.so.6:__memset_sse2_unaligned) redirected to 0x48428e0 (memset)
+--7744-- REDIR: 0x492bec0 (libc.so.6:__memchr_sse2) redirected to 0x4840050 (memchr)
+--7744-- REDIR: 0x4931480 (libc.so.6:__strchr_sse2) redirected to 0x483eb90 (__strchr_sse2)
+--7744-- REDIR: 0x4936b00 (libc.so.6:__strcpy_sse2_unaligned) redirected to 0x483f090 (strcpy)
+--7744-- REDIR: 0x4938140 (libc.so.6:__stpcpy_sse2_unaligned) redirected to 0x4842570 (__stpcpy_sse2_unaligned)
+==7744== 
+==7744== HEAP SUMMARY:
+==7744==     in use at exit: 272 bytes in 1 blocks
+==7744==   total heap usage: 749 allocs, 748 frees, 2,039,642 bytes allocated
+==7744== 
+==7744== Searching for pointers to 1 not-freed blocks
+==7744== Checked 8,477,224 bytes
+==7744== 
+==7744== 272 bytes in 1 blocks are possibly lost in loss record 1 of 1
+==7744==    at 0x483DD99: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
+==7744==    by 0x40149DA: allocate_dtv (dl-tls.c:286)
+==7744==    by 0x40149DA: _dl_allocate_tls (dl-tls.c:532)
+==7744==    by 0x4866322: allocate_stack (allocatestack.c:622)
+==7744==    by 0x4866322: pthread_create@@GLIBC_2.2.5 (pthread_create.c:660)
+==7744==    by 0x10AA74: initTimer (aesdsocket.c:181)
+==7744==    by 0x10B867: main (aesdsocket.c:569)
```

Comment:
This is an actual issue.

Reply:
Is this the Valgrind error for the cleanup of timer thread create?